index.js

```diff
@@ -759,3 +759,28 @@ is.hex = function (value) {
 is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
+
+is.not = negateMethods(is);
+
+function negateMethods(source) {
+	var negatedMethods = {};
+	Object.keys(source).forEach(function (key) {
+		var value;
+    var negatedMethod;
+		if ( source.hasOwnProperty(key) ) {
```

Comment:
Please use the `has` module, or cache `Object.prototype.hasOwnProperty` and `.call` with that